src/viewport/celery_app.py

```
@@ -73,4 +79,66 @@ def create_celery_app(settings: CelerySettings | None = None) -> Celery:
 
 celery_app = create_celery_app()
 
-__all__ = ["celery_app", "create_celery_app", "CelerySettings"]
+_TASK_START_TIMES: dict[str, float] = {}
+_TASK_CONTEXT_TOKENS: dict[str, object] = {}
+_CELERY_SIGNALS_REGISTERED = False
+
+
+def register_celery_observability(app: Celery) -> None:
```

Ruff (0.15.12)

[warning] Unused function argument: `app`

(ARG001)